exercises/pascals-triangle/examples/src/PascalsTriangle.purs

```diff
@@ -0,0 +1,24 @@
+module PascalsTriangle
+  (rows
+  ) where
+
+import Prelude
+import Data.Array (drop, zip, (:))
+import Data.Maybe (Maybe(..))
+import Data.Tuple (uncurry)
+
+type Row = Array Int
+type Mat = Array (Array Int)
+
+rows' :: Row -> Int -> Mat
+rows' row 1 = [row]
```

Comment:
Thanks @paf31 - I knew there was something I could use to make this more readable :)